### ISSUE TITLE
[Story] Show user that media upload fails due to unverified device or changed identity

### ISSUE THREAD

**Issue by mxandreas** [vector-im/element-meta#2574]
### Description

**Background**

When sending a text message, it currently fails to send in case of these 2 scenarios.
* A verified user has an unverified device.
* A verified user's identity has changed.

The message appears on the timeline with a "failure" indicator, and user can tap on it to send it anyway (or additionally withdraw verification). The respective design [[1](https://www.figma.com/design/0oUTCbYhROEDprSp7bdWhx/Identity-Changes?node-id=7-27170&t=3Z8gSwpEOZNmY8Cp-4), [2](https://www.figma.com/design/0oUTCbYhROEDprSp7bdWhx/Identity-Changes?node-id=13-4364&t=3Z8gSwpEOZNmY8Cp-4)] for reference (iOS version).

The challenge is that when a user is trying to send a file, they will not see anything on the timeline. Sending the media just fails with a general error message and the user has no idea what is wrong. Also, besides just knowing what is wrong, the user must also be able to continue messaging.

**Story**

* As a user
* I want to understand why the media fails
* So that I know what is wrong and have also a way to fix it to continue messaging

### Acceptance criteria

- TBD

### Size estimate

None

### Dependencies

- None

### Out of scope

- Nothing

### Open questions

```[tasklist]
### Questions
```

### Subtasks

```[tasklist]
### Android
```

```[tasklist]
### iOS
```

```[tasklist]
### Rust
```

```[tasklist]
### Other
```

**Comment by mxandreas** [comment]
We had a short sync with @americanrefugee and concluded that from the UX perspective it makes the most sense to solve this in the same way as text messages:

![image](https://github.com/user-attachments/assets/0abcf20e-7996-40b1-97bd-dd5a26c67957)

@manuroe could you please confirm that this is not feasible to be done until we have implemented the local echo for file upload. Once the latter is there, it becomes fairly straightforward / easy?

**Comment by manuroe** [comment]
> @manuroe could you please confirm that this is not feasible to be done until we have implemented the local echo for file upload. Once the latter is there, it becomes fairly straightforward / easy?

This is correct. Media do not appear in the timeline until they are received by the server, which will not happen in this case.

 If we want to add such alert now, we need to do it in the current blocking sending flow.
